app/styles/layout/_select-inline.scss

```diff
@@ -204,6 +204,7 @@ $list-height: 220px;
       margin: 5px 7px 0 0;
       position: absolute;
       width: $icon-width;
+      left: 10px;
```

Comment:
Do we have some style guides for CSS? Just alphabetical ordering or maybe like this:

## Declaration Order

Declarations should first be consistent with the rest of the
application. After that it should be in order of what affects height.

`margin`
`padding`
`display`
`position`
`top`
`right`
`bottom`
`left`
`float`
`overflow`
`border`
`border-radius`
`background`
`font`
`line-height`
`color`
`letter-spacing`
`transitions`

If there is a shorthand version for that set of styles, follow in the
order of the shorthand. For `margin` that would be `margin-top`,
`margin-right`, `margin-bottom` then `margin-left`. For `border` that
would be `border-width`, `border-style` then `border-color`.

from: https://github.com/dockyard/styleguides/blob/master/ux-dev/css.md#declaration-order